File: app/src/main/java/com/tukholko/assistant/app/Product.java
package com.tukholko.assistant.app;

import com.google.gson.annotations.SerializedName;

public class Product {
    @SerializedName("Product_name")
    public String name;
    @SerializedName("Product_weight")
    public Double weight;
    @SerializedName("Manufacturer")
    public String manufacturer;
    @SerializedName("Product_description")
    public String description;
    @SerializedName("Product_price")
    public Double price;

    @Override
    public String toString() {
        return "Product{" +
                "name='" + name + '\'' +
                ", weight=" + weight +
                ", manufacturer='" + manufacturer + '\'' +
                ", description='" + description + '\'' +
                ", price=" + price +
                '}';
    }
}

File: app/src/main/java/com/tukholko/assistant/model/Product.kt
package com.tukholko.assistant.model

import com.google.gson.annotations.SerializedName

data class Product(
        @SerializedName("Product_name")
        var name: String,
        @SerializedName("Product_weight")
        var weight: Double,
        @SerializedName("Manufacturer")
        var manufacturer: String,
        @SerializedName("Product_description")
        var description: String,
        @SerializedName("Product_price")
        var price: Double
)
